src/app/services/authenticationService.js

```diff
@@ -0,0 +1,31 @@
+angular.module('app-bootstrap').factory('authenticationService', [
+  'httpService', '$http', 'localStorageService', '$state',
+  function (httpService, $http, localStorageService, $state) {
+
+    return {
+      login: (params) => {
+        const defaultSuccessCb = (data) => {
+          localStorageService.set('session_token', data.data.user.session_token);
```

Comment:
I think we should also set the user in the rootscope and have a method to get the logged user that returns the rootscope user or fetches it from the local storage (when it fetches it from the local storage it should be saved in the rootscope too).